truss/templates/shared/uvicorn_config.py

```diff
@@ -0,0 +1,64 @@
+import asyncio
+import concurrent.futures
+import logging
```

Comment:
is `logging` async compatible?

Reply:
Don't think logging is async compatible, but we don't log much. Theoretically, it could be an issue if the pre and post processing functions in truss do a lot of logging, predict runs synchronously on an executor. I think it's a fairly edge case and unrelated to this change. But good observation, something to take note of.

Reply:
won't it give unpredictable performance? Since the io blocks the event loop - even if it is small?

Reply:
@squidarth let's make a note to look into this as part of the truss server revamp. I think it's worth looking into as a followup